fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -334,7 +395,7 @@ ACTOR Future<Void> getLatestCipherKeys(Reference<EncryptKeyProxyData> ekpProxyDa
 	state std::vector<EKPBaseCipherDetails> cachedCipherDetails;
 	state EKPGetLatestBaseCipherKeysRequest latestKeysReq = req;
 	state EKPGetLatestBaseCipherKeysReply latestCipherReply;
-	state Arena& arena = latestCipherReply.arena;
+	state Arena& arena = latestKeysReq.arena;
```

Comment:
Maybe just a separate arena?
```suggestion
	state Arena arena;
```